### DISCUSSION TITLE
Translate with titile when using nav in file mkdocs.yaml

### DISCUSSION THREAD

**Discussion by SavantEVO** [squidfunk/mkdocs-material#6303] [Q&A]
I need to switch language English and Vietnamese but just the file.md can switch. So the title can not. Please help me how to do that ?
![image](https://github.com/squidfunk/mkdocs-material/assets/109493900/37dcd7dd-6ab3-4fbb-a6aa-8c62b5011b39)

![image](https://github.com/squidfunk/mkdocs-material/assets/109493900/0d888079-fec6-4278-a2ff-4309772d4d63)

**Comment by alexvoss** [comment]
I know of two principal ways to achieve this: 

- have projects side-by side, one for Vietnamese, one for English. You can have most of the settings in a `mkdocs.yml` file in the parent folder and use [configuration inheritance](https://www.mkdocs.org/user-guide/configuration/#configuration-inheritance) to get them imported for each language version. If you are a sponsor, you could give the [projects plugin](https://squidfunk.github.io/mkdocs-material/plugins/projects/) a try, which would allow you to build/serve the whole website with one command.
- you can use the [static i18n plugin](https://github.com/ultrabug/mkdocs-static-i18n/), which takes a different approach. With this, you would have only one `mkdocs.yml` file in which you can define translations for your navigation. The translations of the pages lie next to each other and have filenames that include the language, e.g., `index.en.md` and `index.vi.md`.

There may be other options but I think these are the commonly used ones.

**Reply by SavantEVO** [reply]
Oh thank you so much. I done 
plugins:
  - search
  - mkdocs-video

  - i18n:
      docs_structure: suffix
      fallback_to_default: true
      reconfigure_material: true
      reconfigure_search: true
      languages:
        - locale: en
          default : True
          name: English
          build: true
        - locale: vi
          name: Vietnamese
          build: true
